### ISSUE TITLE
Structural variants as input to sciclone

### ISSUE THREAD

**Issue by phdstudent-oje** [genome/sciclone#48]
Dear @chrisamiller,

Hi, I'm Oh.
If I have all information about structural variants (alt reads, ref reads, and vaf), can I use it as input to sciclone?
Many thanks.
Oh.

**Comment by phdstudent-oje** [comment]
Dear @chrisamiller,
Thanks for your prompt reply. 

The structural variation I mentioned in the first question refers to the integration event.
I have all the information of the integration event (alt reads, ref reads, and vaf).
This information can be converted into the form of a point mutation. 

_Q. I wonder if it is okay to use these data as a **variant input** of sciclone._

Will your comment on biostar be the answer?
"The short answer is yes, you can convert any kind of measure to a VAF-equivalent and use it."

Many thanks. 
Oh.

**Comment by chrisamiller** [comment]
Yes, you can use that event in the same way you would use the VAF of a point mutation
